http-gateway/web/src/components/user-widget/user-widget.js

```diff
@@ -1,35 +1,45 @@
 import { useRef, useState, memo } from 'react'
 import classNames from 'classnames'
-import { useAuth0 } from '@auth0/auth0-react'
 import { useIntl } from 'react-intl'
-
+import Avatar from 'react-avatar'
 import { useClickOutside } from '@/common/hooks'
 import { messages as t } from './user-widget-i18n'
 import './user-widget.scss'
+import { useAuth } from 'oidc-react'
 
 export const UserWidget = memo(() => {
   const [expanded, setExpand] = useState(false)
-  const { isLoading, isAuthenticated, user, logout } = useAuth0()
+  const { isLoading, userData, signOutRedirect } = useAuth()
   const { formatMessage: _ } = useIntl()
   const clickRef = useRef()
   useClickOutside(clickRef, () => setExpand(false))
 
-  if (!isLoading && !isAuthenticated) {
+  if (!isLoading && !userData) {
     return null
   }
 
   return (
     <div id="user-widget" className="status-bar-item" ref={clickRef}>
       <div className="toggle" onClick={() => setExpand(!expanded)}>
         <div className="user-avatar">
-          <img src={user.picture} alt="User Avatar" />
```

Comment:
If user picture is present, why not to display it? You can check if it's available by calling https://authts.github.io/oidc-client-ts/interfaces/OidcStandardClaims.html#picture . If it's present you can use it.